tests/workflows/test_image_analysis.py

```diff
@@ -0,0 +1,30 @@
+import dask
+import dask.array as da
+import numpy as np
+from dask_image import ndfilters, ndmeasure, ndmorph
+
+
+def test_BBBC039(small_client):
+    images = da.from_zarr(
+        "s3://coiled-datasets/BBBC039", storage_options={"anon": True}
+    )
+    smoothed = ndfilters.gaussian_filter(images, sigma=[0, 1, 1])
+    thresh = ndfilters.threshold_local(smoothed, block_size=images.chunksize)
+    threshold_images = smoothed > thresh
+    structuring_element = np.array(
```

Comment:
Suggest adding a short comment: to this line about why we're not using the default structuring element. Eg:
```python
# Since this image stack appears to be 3-dimensional,
# we sandwich a 2d structuring element in between zeros
# so that each 2d image slice has the binary closing applied independently
```
Apparently I only ever said that verbally during the talk, may as well write it down.